dictsource/be_rules

```diff
@@ -36,35 +36,35 @@
        г       Q     // ɣ
 
 .group д
-       д       d[
-       дз      dz  // d̻͡z̪
-       ддз     dz:
-       дж      dz. // ɖ͡ʐ
+       д       d
+       дз      dz    // fix IPA conversion to d͡z
+//     ддз           // doesn't work
+       дж      dz.   // fix IPA conversion to ɖ͡ʐ
 
 .group е
        е       ;E    // ʲɛ
-    _) е       jE
-    ') е       jE
+//  _) е       jE    // applies not only to start of the word
+    ') е       ;jE
     ь) е       jE
-  L01) е       jE
+  L01) е       jE    // doesn't work with: е, ё, і, о, у, ы, э, я
 
 .group ё
-       ё       ;'O    // ʲɔ
-    _) ё       j'O
-    ') ё       j'O
-    ь) ё       j'O
-  L01) ё       j'O
+       ё       ;O    // ʲɔ
+ // _) ё       jO    // applies not only to start of the word
+    ') ё       ;jO
```

Comment:
you swapped the behavior for apostrophe and soft sign

Reply:
fixed